icefoganalytics/elcc-data-management
# Fix Mssql Server Health Check Broken by Image Update

Relates to:

- https://yg-hpw.atlassian.net/browse/ELCC-66

# Context

https://hub.docker.com/r/microsoft/mssql-server/ release a breaking change in their image.
Since our image wasn’t locked to a specific version, this issue went unnoticed until I tried to clean boot the project.

# Implementation

1. Remove the health check from the container level, and rely on the JS level initializers.
2. Re-write local initializer code with patterns from from https://github.com/icefoganalytics/wrap/pull/86.
3. Use fixed MSSQL version, to avoid having this problem occur in the future.


# Testing Instructions

1. Drop the database via `dev down -v`
2. Boo the app via `dev up`
4. Check that the app complies, and that you can log in at http://localhost:8080.
